rasa/utils/tensorflow/models.py

```diff
@@ -341,8 +360,10 @@ def load(
             evaluate_on_num_examples=0,
             batch_strategy=SEQUENCE,
             silent=True,  # don't confuse users with training output
-            loading=True,  # don't use tensorboard while loading
-            eager=True,  # no need to build tf graph, eager is faster here
+            loading=True,  # don't use tensorboard when doing a dummy fit run
+            eager=False
+            if finetune_mode
+            else True,  # load in eager mode only for prediction phase
```

Comment:
not sure if `black` will complain, but parenthesis make it easier to spot that this is all related to `eager=`
```suggestion
            eager=(False
                if finetune_mode
                else True
            ),  # load in eager mode only for prediction phase
```